scripts/install_zds.sh

```diff
@@ -41,7 +41,7 @@ fi
 
 # os-specific package install
 if  ! $(_in "-packages" $@) && ( $(_in "+packages" $@) || $(_in "+base" $@) || $(_in "+full" $@) ); then
-    echo "* [+packages] installing packages (require sudo)"
+    echo "* [+packages] installing packages (require sudo installed)"
```

Comment:
en fait "require to be run as super user"

Reply:
bah non justement , make install-linux il faut le lancer en normal non ?

Reply:
Absolument. C'est juste que dans le script, y'a un sudo

Reply:
si tu veux une phrase encore plus claire "some subcommands will be run as super-user"